_includes/specials/taproot/en/21-thanks.md

```diff
@@ -0,0 +1,509 @@
+Natanael,
+Oleg Andreev,
+Pavol Rusnak,
+Pieter Wuille,
+Prayank,
+R E Broadley,
+Riccardo Casatta,
+Robert Spigler,
+Ruben Somsen,
+Russell O'Connor,
+Rusty Russell,
+Ryan Grant,
+Salvatore Ingala,
+Samson Mow,
+Sjors Provoost,
+Steve Lee,
+Tamas Blummer,
+Thomas Hartman,
+Tim Ruffing,
+Vincent Truong,
+vjudeu,
+yancy,
+yanmaani---,
+and
+ZmnSCPxj.
+</i>
+
+However, many of the ideas included in taproot, such as [schnorr
+signatures][topic schnorr signatures] and [MAST][topic mast], predate
+taproot by years or even decades.  It's beyond our capacity to list the
+many contributors to those ideas, but we owe them our thanks
+nonetheless.
+
+{:#taproot-bip-review}
+**Taproot BIP review**
+
+Starting in November 2019, a large number of users and developers
+participated in an [organized review][news69 review] of taproot and
+related developments.
+
+<!--
+wget -mirror https://gnusha.org/taproot-bip-review/
+cat *.log | sed 's/>//g; s/<//' | awk '{print $2}' | sed 's/_$//' | sort -u
+
+Removed some obvious duplicates and bots -harding
+-->
+
+<i>
+achow101,
+afk11,
+aj,
+alec,
+amiti,
+_andrewtoth,
+andytoshi,
+ariard,
+arik,
+b10c,
+belcher,
+bjarnem,
+BlueMatt,
+bsm1175321,
+cdecker,
+chm-diederichs,
+Chris_Stewart_5,
+cle1408,
+CubicEarth,
+Day,
+ddustin,
+devrandom,
+digi_james,
+dr-orlovsky,
+dustinwinski,
+elichai2,
+evoskuil,
+fanquake,
+felixweis,
+fjahr,
+ghost43,
+ghosthell,
+gmaxwell,
+harding,
+hebasto,
+instagibbs,
+jeremyrubin,
+jnewbery,
+jonatack,
+justinmoon,
+kabaum,
+kanzure,
+luke-jr,
+maaku,
+mattleon,
+michaelfolkson,
+midnight,
+mol,
+Moller40,
+moneyball,
+murch,
+nickler,
+nothingmuch,
+orfeas,
+pinheadmz,
+pizzafrank13,
+potatoe_face,
+pyskell,
+pyskl,
+queip,
+r251d,
+raj_149,
+real_or_random,
+robert_spigler,
+roconnor,
+sanket1729,
+schmidty,
+sipa,
+soju,
+sosthene,
+stortz,
+taky,
+t-bast,
+theStack,
+Tibo,
+waxwing,
+xoyi-,
+and
+ZmnSCPxj.
+</i>
+
+{:#github-prs}
+**GitHub pull requests**
+
+The main implementation of taproot in Bitcoin Core was submitted for
+review starting in January 2020 in [two][bitcoin core #17977] pull
+[requests][bitcoin core #19953].  The following people left a GitHub
+review on those PRs.  <!-- in addition to sipa, who opened the PRs -->
+
+<i>
+Andrew Chow (achow101),
+Anthony Towns (ajtowns),
+Antoine Riard (ariard),
+Ben Carman (benthecarman),
+Ben Woosley (Empact),
+Bram (brmdbr),
+Cory Fields (theuni),
+Dmitry Petukhov (dgpv),
+Elichai Turkel (elichai),
+Fabian Jahr (fjahr),
+flack
+Gregory Maxwell (gmaxwell),
+Gregory Sanders (instagibbs),
+James O'Beirne (jamesob),
+Janus Troelsen (ysangkok),
+Jeremy Rubin (JeremyRubin),
+João Barbosa (promag),
+John Newbery (jnewbery),
+Jon Atack (jonatack),
+Jonathan Underwood (junderw),
+kallewoof,
+Kanon (decryp2kanon),
+kiminuo,
+Luke Dashjr (luke-jr),
+Marco Falke (MarcoFalke),
+Martin Habovštiak (Kixunil),
+Matthew Zipkin (pinheadmz),
+Max Hillebrand (MaxHillebrand),
+Michael Folkson (michaelfolkson),
+Michael Ford (fanquake),
+nopara73,
```

Comment:
```suggestion
Adam Ficsor (nopara73),
```

(from https://youtu.be/QiySI4-MWww?t=11, linked from https://gist.github.com/nopara73)